### ISSUE TITLE
[JOSS] Editor review of ESPnet-SE++ paper

### ISSUE THREAD

**Issue by faroit** [espnet/espnet#5399]
@neillu23 this is another short review of the JOSS submission https://github.com/openjournals/joss-reviews/issues/5403

I did another read of the paper. Can you please address a few more smaller things:

* [x] the paper mentions that espnet-se++ adds "several new models, loss functions,
and training recipes". Even though the interspeech paper is cited, can you add references to the actual model architecture papers as well, if available?
* [x] almost all references lack proper DOIs. This is required for a JOSS publication
* [x] `title={Onssen: an open-source ...}` should be `title={{ONSSEN}: an open-source ...}`

thanks!
